doc/getting-started.md

```diff
@@ -44,10 +44,79 @@ You now have four servers, running on localhost:31410, 31430, 31450 and 31470. T
 
 You are now done. Everything below this line is extra information for the incurably curious.
 
-### important notes for the admin server > widget management pages
+### Important notes for the admin server > widget management pages
 Each widget management page shows a preview of how the widget will look given a set of configurations.
 To make this work however, one must first use the command "npm run build" in the specific widget (in the packages > [widget] folder). This is only neccesary in development. In production these widgets are already prebuild.
 
+### Getting the widget preview to work:
+There are quite a view thinks to take into account when you try to get a new widget to render in the preview component. The following steps need to be done
```

Comment:
There are a few steps to follow when trying to get a new widget rendered for your preview component: